ee/dtr/admin/install/index.md

```diff
@@ -155,21 +128,16 @@ To add replicas to a DTR cluster, use the [join](/reference/dtr/2.7/cli/join/) c
       --ucp-insecure-tls
     ```
 
-    > --ucp-node
+    > Important
     >
-    > The `<ucp-node-name>` following the `--ucp-node` flag is the target node to
-    > install the DTR replica. This is NOT the UCP Manager URL.
+    > The `<ucp-node-name>` following the `--ucp-node` flag is the target node on which to install the DTR replica. It is NOT the UCP Manager URL.
     {: .important}
 
-    When you join a replica to a DTR cluster, you need to specify the
-    ID of a replica that is already part of the cluster. You can find an
-    existing replica ID by going to the **Shared Resources > Stacks** page on UCP.
-    
-3. Check that all replicas are running.
+    When joining a replica to a DTR cluster, it is necessary to specify the
+    ID of a replica that is already part of the cluster. Locate an
+    existing replica ID on the **Shared Resources > Stacks** UCP page.
 
-    In your browser, navigate to UCP's
-    web interface. Select **Shared Resources > Stacks**. All replicas should
-    be displayed.
+3. Confirm that all replicas are running by navigating to UCP's web interface and selecting **Shared Resources > Stacks**. All replicas should display.
```

Comment:
"navigating to the UCP web interface..."
"All replicas should be visible."

Reply:
Not a fan of 'visible' in this context. Instead, I changed "should display" to "should be on display".